app/assets/stylesheets/components/_search_form.scss

```diff
@@ -26,3 +26,22 @@
   }
 }
 
+#autocomplete-suggestions {
+  font-size: 14px;
+  position: absolute;
+  top: calc(100% + 5px);
+  left: 0;
+  width: 100%;
+  background: #FFF;
+  color: #000;
+  border-radius: 3px;
+  box-shadow: 0px 0px 1px rgba(0,0,0,0.2);
+  z-index: 2;
+}
+
```

Comment:
add a:hover no text decoration background use brand green background for light green hovers